src/fsharp/CompileOps.fs

```diff
@@ -3938,6 +3939,7 @@ and [<Sealed>] TcImports(tcConfigP: TcConfigProvider, initialResolutions: TcAsse
     let mutable dllTable: NameMap<ImportedBinary> = NameMap.empty
     let mutable ccuInfos: ImportedAssembly list = []
     let mutable ccuTable: NameMap<ImportedAssembly> = NameMap.empty
+    let mutable ccuThunks = new ConcurrentDictionary<CcuThunk, (unit -> unit)>()
```

Comment:
Could you put a `///` comment on this please, saying what the `unit -> unit` does?

Reply:
Yes, thats a good idea.